README.md

```diff
@@ -33,6 +33,7 @@ Here is the Amphion v0.1 demo, whose voice, audio effects, and singing voice are
 )
 
 ## 🚀 News
+- **2024/6/17**: Amphion has a new release for its VALLE models, it uses Llama as its underlying architecture, and has better model performance, faster training speed, and more readable codes compared to our first version. [![readme](https://img.shields.io/badge/README-Key%20Features-blue)](egs/tts/valle_v2/README.md)
```

Comment:
"VALLE" or "Vall-E"? We'd better stay consistent for naming in the whole repo.

Reply:
renamed